tools/ci_build/github/azure-pipelines/android-arm64-crosscompile-ci-pipeline.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   steps:
     - template: templates/set-test-data-variables-step.yml
 
-    - script: 'tools/ci_build/github/linux/run_dockerbuild.sh -o android -r $(Build.BinariesDirectory) -d cpu'
+    - script: 'tools/ci_build/github/linux/run_dockerbuild.sh -o android -r $(Build.BinariesDirectory) -d cpu -x "--build_wheel"'
```

Comment:
build_wheel can be left off for arm cross compile I think.

Reply:
Done